hazelcast/src/main/java/com/hazelcast/spi/SplitBrainMergeEntryView.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi;
+
+import com.hazelcast.core.EntryView;
+
+/**
+ * Entry info for merging split-brain sub-clusters.
+ *
+ * @param <K> the type of the key
+ * @param <V> the type of the value
+ */
+public interface SplitBrainMergeEntryView<K, V> extends EntryView<K, V> {
```

Comment:
I don't see a reason to repeat (`@Override`) the methods from `EntryView` interface. Is there any?

Reply:
Ah yes, the reason is that I wasn't sure if it's a good design choice to extend `EntryView` or not. So I started this as experiment have the methods duplicated in the interface, so we could easily remove the `extends EntryView` if needed. But I'm still not sure if this would break any compatibility.

So if we stick with the inheritance, yes, we should probably remove the overrides.

Reply:
I was thinking about removing `EntryView` from the class, since it's a dependency on the `core` package etc. But you are right, this won't work. So I removed the duplicated methods and we'll stick with that inheritance for HZ 3.x :)